compose/container.py

```diff
@@ -45,6 +45,14 @@ def create(cls, client, **options):
         response = client.create_container(**options)
         return cls.from_id(client, response['Id'])
 
+    @classmethod
+    def exec_command(cls, client, c_id, command, **options):
+        detach = options.get("detach")
```

Comment:
I think this is a problem. You can't really really attach to a service because it represents multiple containers.